docs/run.js

```diff
@@ -35,7 +35,7 @@ function blocks(filepath) {
     }
 
     const wrap = str => {
-      return `(function (exports, console, require, module, __filename, __dirname) { ${str}\n});`
+      return `(function (exports, console, require, module, __filename, __dirname) { ${str}\n});`;
```

Comment:
These were generated by running: `eslint fix` with the eslint configuration from the repository. 
I hope it's OK.